### DISCUSSION TITLE
Rearrange christmas backlighting for taiko and mania beatmaps

### DISCUSSION THREAD

**Discussion by I21b** [ppy/osu#31247] [Ideas] [answered]
|osu!|osu!taiko|osu!catch|osu!mania4k|osu!mania7k|
|---|---|---|---|---|
|![osu](https://github.com/user-attachments/assets/3893a5c7-6c55-49fd-9672-8b3c6d0bdfb1)|![taiko](https://github.com/user-attachments/assets/10dc2487-8b28-4864-bd23-b3dedc4ec10a)|![catch](https://github.com/user-attachments/assets/b84e2b52-bea4-4474-b0ce-e35846454a42)|![mania4](https://github.com/user-attachments/assets/cadcaac5-cb1a-4742-a91a-a3a7658dfb12)|![mania7](https://github.com/user-attachments/assets/5014b583-c787-4ce8-a4d8-a1f11539f6d7)|
|👍|![taikofull](https://github.com/user-attachments/assets/7ed989fa-605b-4941-9ef9-599e2c1e002d)|👍|![mania4full](https://github.com/user-attachments/assets/5fc66e2a-e78e-407d-89b4-c3aeb6236c24)|![mania7full](https://github.com/user-attachments/assets/ee772e90-de9c-4ea4-a941-b703746b7d39)|

**Comment by I21b** [comment]
Should I @ peppy here?

**Reply by peppy** [reply, marked as answer]
I was aware of this. If you want to provide a pull request I'll consider it.
